docs/pages/database-access/guides/rds.mdx

```diff
@@ -209,6 +209,8 @@ $ tsh db logout postgres-rds
 
 (!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
 
+(!docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx!)
```

Comment:
Why not just include this topic in aws-troubleshooting.mdx?

Reply:
This error is not applicable to other AWS databases (e.g. AWS keyspaces, Elaticache, etc.). I think it's cleaner this way.